core/src/components/select/select.ios.scss

```diff
@@ -4,6 +4,10 @@
 // iOS Select
 // --------------------------------------------------
 
+:host {
+  --highlight-height: 0px;
```

Comment:
This doesn't need to be defined since the highlight is defined in `md` but I kept it here in case we ever move the highlight to the base file.